tests/fixtures/doc_auth_verify.html

```diff
@@ -0,0 +1,193 @@
+<!--
+  This is a sample response from the IdP running on localhost.
+  The repsonse is from /verify/doc_auth/verify page.
+
+  This response has 3 different authhenticity_tokens
```

Comment:
nit, typo s/authhenticity_tokens/authenticity_tokens/

Reply:
ugh, worth pointing out.  I mis-spelled this once last week and it cost me 30 min of head-scratching.  Thanks.